[2023-05-10T02:13:45.234+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrape_and_save_to_csv.scrape_teams_25-30 scheduled__2022-05-06T00:00:00+00:00 [queued]>
[2023-05-10T02:13:45.256+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrape_and_save_to_csv.scrape_teams_25-30 scheduled__2022-05-06T00:00:00+00:00 [queued]>
[2023-05-10T02:13:45.257+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-10T02:13:45.259+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-10T02:13:45.259+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-10T02:13:45.278+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): scrape_teams_25-30> on 2022-05-06 00:00:00+00:00
[2023-05-10T02:13:45.284+0000] {standard_task_runner.py:55} INFO - Started process 17257 to run task
[2023-05-10T02:13:45.290+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'scrape_and_save_to_csv', 'scrape_teams_25-30', 'scheduled__2022-05-06T00:00:00+00:00', '--job-id', '4005', '--raw', '--subdir', 'DAGS_FOLDER/scrape_and_push_dag.py', '--cfg-path', '/tmp/tmpigb50uxv']
[2023-05-10T02:13:45.293+0000] {standard_task_runner.py:83} INFO - Job 4005: Subtask scrape_teams_25-30
[2023-05-10T02:13:45.411+0000] {task_command.py:389} INFO - Running <TaskInstance: scrape_and_save_to_csv.scrape_teams_25-30 scheduled__2022-05-06T00:00:00+00:00 [running]> on host f41dbfa3d325
[2023-05-10T02:13:45.450+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-05-10T02:13:45.499+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=scrape_and_save_to_csv
AIRFLOW_CTX_TASK_ID=scrape_teams_25-30
AIRFLOW_CTX_EXECUTION_DATE=2022-05-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-06T00:00:00+00:00
[2023-05-10T02:13:47.034+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:47.048+0000] {helpers.py:99} ERROR - Retrying! 9 attempts before skipping!
[2023-05-10T02:13:49.037+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:49.038+0000] {helpers.py:99} ERROR - Retrying! 8 attempts before skipping!
[2023-05-10T02:13:50.679+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:50.680+0000] {helpers.py:99} ERROR - Retrying! 7 attempts before skipping!
[2023-05-10T02:13:52.446+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:52.448+0000] {helpers.py:99} ERROR - Retrying! 6 attempts before skipping!
[2023-05-10T02:13:54.305+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:54.308+0000] {helpers.py:99} ERROR - Retrying! 5 attempts before skipping!
[2023-05-10T02:13:55.966+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:55.973+0000] {helpers.py:99} ERROR - Retrying! 4 attempts before skipping!
[2023-05-10T02:13:57.710+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:57.711+0000] {helpers.py:99} ERROR - Retrying! 3 attempts before skipping!
[2023-05-10T02:13:59.389+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:13:59.390+0000] {helpers.py:99} ERROR - Retrying! 2 attempts before skipping!
[2023-05-10T02:14:01.125+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:01.128+0000] {helpers.py:99} ERROR - Retrying! 1 attempts before skipping!
[2023-05-10T02:14:02.933+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:02.934+0000] {helpers.py:99} ERROR - Retrying! 0 attempts before skipping!
[2023-05-10T02:14:04.652+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:04.653+0000] {helpers.py:99} ERROR - Retrying! -1 attempts before skipping!
[2023-05-10T02:14:06.637+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:06.639+0000] {helpers.py:99} ERROR - Retrying! 9 attempts before skipping!
[2023-05-10T02:14:09.813+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:09.817+0000] {helpers.py:99} ERROR - Retrying! 8 attempts before skipping!
[2023-05-10T02:14:12.970+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:12.973+0000] {helpers.py:99} ERROR - Retrying! 7 attempts before skipping!
[2023-05-10T02:14:16.904+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:16.907+0000] {helpers.py:99} ERROR - Retrying! 6 attempts before skipping!
[2023-05-10T02:14:20.598+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:20.600+0000] {helpers.py:99} ERROR - Retrying! 5 attempts before skipping!
[2023-05-10T02:14:24.709+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:24.710+0000] {helpers.py:99} ERROR - Retrying! 4 attempts before skipping!
[2023-05-10T02:14:28.561+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:28.571+0000] {helpers.py:99} ERROR - Retrying! 3 attempts before skipping!
[2023-05-10T02:14:32.748+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:32.749+0000] {helpers.py:99} ERROR - Retrying! 2 attempts before skipping!
[2023-05-10T02:14:37.007+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:37.010+0000] {helpers.py:99} ERROR - Retrying! 1 attempts before skipping!
[2023-05-10T02:14:41.216+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:41.217+0000] {helpers.py:99} ERROR - Retrying! 0 attempts before skipping!
[2023-05-10T02:14:45.063+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:45.067+0000] {helpers.py:99} ERROR - Retrying! -1 attempts before skipping!
[2023-05-10T02:14:48.816+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:48.820+0000] {helpers.py:99} ERROR - Retrying! 9 attempts before skipping!
[2023-05-10T02:14:52.489+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:52.491+0000] {helpers.py:99} ERROR - Retrying! 8 attempts before skipping!
[2023-05-10T02:14:56.761+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:14:56.762+0000] {helpers.py:99} ERROR - Retrying! 7 attempts before skipping!
[2023-05-10T02:15:01.193+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:01.195+0000] {helpers.py:99} ERROR - Retrying! 6 attempts before skipping!
[2023-05-10T02:15:05.154+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:05.157+0000] {helpers.py:99} ERROR - Retrying! 5 attempts before skipping!
[2023-05-10T02:15:08.899+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:08.900+0000] {helpers.py:99} ERROR - Retrying! 4 attempts before skipping!
[2023-05-10T02:15:12.633+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:12.636+0000] {helpers.py:99} ERROR - Retrying! 3 attempts before skipping!
[2023-05-10T02:15:16.664+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:16.665+0000] {helpers.py:99} ERROR - Retrying! 2 attempts before skipping!
[2023-05-10T02:15:20.850+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:20.852+0000] {helpers.py:99} ERROR - Retrying! 1 attempts before skipping!
[2023-05-10T02:15:24.597+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:24.600+0000] {helpers.py:99} ERROR - Retrying! 0 attempts before skipping!
[2023-05-10T02:15:28.915+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:28.917+0000] {helpers.py:99} ERROR - Retrying! -1 attempts before skipping!
[2023-05-10T02:15:32.771+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:32.774+0000] {helpers.py:99} ERROR - Retrying! 9 attempts before skipping!
[2023-05-10T02:15:36.443+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:36.445+0000] {helpers.py:99} ERROR - Retrying! 8 attempts before skipping!
[2023-05-10T02:15:40.654+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:40.656+0000] {helpers.py:99} ERROR - Retrying! 7 attempts before skipping!
[2023-05-10T02:15:44.497+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:44.499+0000] {helpers.py:99} ERROR - Retrying! 6 attempts before skipping!
[2023-05-10T02:15:48.706+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:48.710+0000] {helpers.py:99} ERROR - Retrying! 5 attempts before skipping!
[2023-05-10T02:15:52.442+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:52.444+0000] {helpers.py:99} ERROR - Retrying! 4 attempts before skipping!
[2023-05-10T02:15:56.693+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:15:56.699+0000] {helpers.py:99} ERROR - Retrying! 3 attempts before skipping!
[2023-05-10T02:16:00.772+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:00.775+0000] {helpers.py:99} ERROR - Retrying! 2 attempts before skipping!
[2023-05-10T02:16:05.025+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:05.028+0000] {helpers.py:99} ERROR - Retrying! 1 attempts before skipping!
[2023-05-10T02:16:08.660+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:08.662+0000] {helpers.py:99} ERROR - Retrying! 0 attempts before skipping!
[2023-05-10T02:16:12.440+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:12.442+0000] {helpers.py:99} ERROR - Retrying! -1 attempts before skipping!
[2023-05-10T02:16:16.734+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:16.737+0000] {helpers.py:99} ERROR - Retrying! 9 attempts before skipping!
[2023-05-10T02:16:20.394+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:20.399+0000] {helpers.py:99} ERROR - Retrying! 8 attempts before skipping!
[2023-05-10T02:16:24.526+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:24.528+0000] {helpers.py:99} ERROR - Retrying! 7 attempts before skipping!
[2023-05-10T02:16:28.264+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:28.266+0000] {helpers.py:99} ERROR - Retrying! 6 attempts before skipping!
[2023-05-10T02:16:32.556+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:32.569+0000] {helpers.py:99} ERROR - Retrying! 5 attempts before skipping!
[2023-05-10T02:16:36.688+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:36.690+0000] {helpers.py:99} ERROR - Retrying! 4 attempts before skipping!
[2023-05-10T02:16:40.383+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:40.385+0000] {helpers.py:99} ERROR - Retrying! 3 attempts before skipping!
[2023-05-10T02:16:44.561+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:44.562+0000] {helpers.py:99} ERROR - Retrying! 2 attempts before skipping!
[2023-05-10T02:16:49.007+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:49.009+0000] {helpers.py:99} ERROR - Retrying! 1 attempts before skipping!
[2023-05-10T02:16:52.718+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:52.720+0000] {helpers.py:99} ERROR - Retrying! 0 attempts before skipping!
[2023-05-10T02:16:56.529+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:16:56.533+0000] {helpers.py:99} ERROR - Retrying! -1 attempts before skipping!
[2023-05-10T02:17:00.878+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:17:00.879+0000] {helpers.py:99} ERROR - Retrying! 9 attempts before skipping!
[2023-05-10T02:17:04.596+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:17:04.598+0000] {helpers.py:99} ERROR - Retrying! 8 attempts before skipping!
[2023-05-10T02:17:08.777+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:17:08.779+0000] {helpers.py:99} ERROR - Retrying! 7 attempts before skipping!
[2023-05-10T02:17:12.526+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:17:12.529+0000] {helpers.py:99} ERROR - Retrying! 6 attempts before skipping!
[2023-05-10T02:17:17.138+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:17:17.140+0000] {helpers.py:99} ERROR - Retrying! 5 attempts before skipping!
[2023-05-10T02:17:21.901+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:17:21.904+0000] {helpers.py:99} ERROR - Retrying! 4 attempts before skipping!
[2023-05-10T02:17:26.895+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:17:26.896+0000] {helpers.py:99} ERROR - Retrying! 3 attempts before skipping!
[2023-05-10T02:17:31.654+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:17:31.657+0000] {helpers.py:99} ERROR - Retrying! 2 attempts before skipping!
[2023-05-10T02:17:36.375+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:17:36.376+0000] {helpers.py:99} ERROR - Retrying! 1 attempts before skipping!
[2023-05-10T02:17:41.718+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:17:41.721+0000] {helpers.py:99} ERROR - Retrying! 0 attempts before skipping!
[2023-05-10T02:17:46.636+0000] {helpers.py:98} ERROR - type object 'datetime.datetime' has no attribute 'datetime'
[2023-05-10T02:17:46.638+0000] {helpers.py:99} ERROR - Retrying! -1 attempts before skipping!
[2023-05-10T02:17:47.662+0000] {helpers.py:108} INFO - Saved in /opt/***/data/output_scrape_teams_25-30.csv
[2023-05-10T02:17:47.664+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-10T02:17:47.683+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=scrape_and_save_to_csv, task_id=scrape_teams_25-30, execution_date=20220506T000000, start_date=20230510T021345, end_date=20230510T021747
[2023-05-10T02:17:47.762+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-05-10T02:17:47.793+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
